test/unit/Git/PickLastMinorVersionFromCollectionTest.php

```diff
@@ -28,6 +28,7 @@ public function lastStableMinorVersionForCollectionProvider() : array
             ['2.2.0', ['1.2.0', '2.2.0', '2.2.1', '1.2.1']],
             ['2.2.0', ['1.2.0', '2.2.0', '2.2.0-alpha1', '2.2.1', '1.2.1']],
             ['2.2.0', ['1.2.0', '2.2.0-alpha1', '2.2.0', '2.2.1', '1.2.1']],
+            ['2.6.0', ['1.1.1', '3.0.0-alpha1', '2.7.0-beta1', '2.6.2', '2.0.0', '2.6.1', '2.6.0']],
```

Comment:
Shouldn't this be `2.6.2`?

Reply:
I've explicitly changed it in https://github.com/Roave/BackwardCompatibilityCheck/pull/118/commits/33e851fd00086a77f053e3d80132402dd31e8bd7 (see commit message)

Reply:
Sorry, didn't get the "lastMinorVersion" thingy....

Reply:
That's alright - it was a design decision :-)